ts/packages/cli/src/base.ts

```diff
@@ -76,6 +76,19 @@ export abstract class BaseCommand<T extends typeof Command> extends Command {
     ux.action.stop(` ${mark} ${chalk.dim(text ?? defaultText)}`);
   }
 
+  getCacheStoreInstanceByStoreFlag(storeFlag?: string) {
+    if (!storeFlag) {
+      throw new Error('--store flag is missing');
+    }
+
+    const storeCache = this.config.cli.data.stores?.[storeFlag]?.cache;
+    if (!storeCache) {
+      return false;
```

Comment:
NIT: i think `undefined` is more accurate here rather than `false`

Reply:
I don't. It's ok that you dont have to set it to true or false in the configuration file but the program eventually wants to know if it's true or false.

anyway, changed to you suggestion.